tests/v2_0/Api/FamilyVariant/GetFamilyVariantApiIntegration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Akeneo\Pim\tests\v2_0\Api\FamilyVariant;
+
+use Akeneo\Pim\tests\Common\Api\ApiTestCase;
+
+class GetFamilyVariantApiIntegration extends ApiTestCase
+{
+    public function testGetFamilyVariant()
```

Comment:
test to find a family variant which does not exist please (same for product model)

Reply:
done